pyi.Dockerfile

```diff
@@ -269,6 +255,18 @@ RUN if grep -q "CentOS Linux 8" /etc/os-release ; then \
     fi
 RUN python3 -m pip install --no-cache-dir -r exe-requirements.txt
 
+FROM ${NODE_PACKAGE_BUILDER_GLIBC} as node-package-builder-glibc
```

Comment:
This can happen earlier - it only requires the installation of Python, not the installation of our `requirements.txt` etc.

I suppose it can be placed installed after
```
# we want the latest pip
# hadolint ignore=DL3013
RUN python3 -m pip install --no-cache-dir --upgrade pip
```

Strive to place things as earlier as possible in Dockerfiles, to avoid unnecessary coupling. In the current state, if I modify e.g the `requirements.txt`, then this thing will be rebuilt unnecessarily